src/chrono.rs

```diff
@@ -0,0 +1,26 @@
+use crate::Rtc;
+use chrono::{DateTime, TimeZone as _, Utc};
+use core::num::TryFromIntError;
+
+impl Rtc {
+    /// Returns the current time.
+    pub fn get_time(&self) -> DateTime<Utc> {
```

Comment:
It is better to add `#doc[cfg(feature = "chrono")]` to generated more readable docs.

Reply:
Unfortunately `#[doc(cfg(...))]` is still unstable: https://doc.rust-lang.org/nightly/unstable-book/language-features/doc-cfg.html